client/question/domain/ErrorTracebackObjectFactory.js

```diff
@@ -143,6 +143,10 @@ tie.factory('ErrorTracebackObjectFactory', [
       // Split the error string by newlines for easier processing.
       var splitErrorMessage = pythonError.trim().split(/\r?\n/);
       var errorMessage = splitErrorMessage[splitErrorMessage.length - 1];
+      if (splitErrorMessage.length === 1) {
```

Comment:
This seems like a rather indirect way of figuring out whether there's a line number. In particular, is it possible that multi-line messages might have no line number? If so then that means this check is insufficient.

One suggestion is to check for the presence of LINE_DELIMITER_PYTHON in the error message -- or, more robustly, use a regex to check for things like ` line \d+ ` to prevent false positives.

Reply:
I can do that.

Reply:
That should be a bit better.